webapp/channels/src/components/admin_console/permission_schemes_settings/__snapshots__/permission_description.test.tsx.snap

```diff
@@ -69,7 +69,7 @@ exports[`components/admin_console/permission_schemes_settings/permission_descrip
             <span>
               Inherited from 
               <a
-                key=".$.1"
+                key="1/.1"
```

Comment:
This is due to react-intl upgrade from 6.6.2 to 7.1.14. Note that the workspace packages declare "react-intl": "*", which defers to the lockfile. When I regenerated package-lock.json for Node 24, npm resolved `*` to the latest version. I now pinned the explicit version in webapp/package.json to make future upgrades intentional and observable but remain `*` on packages so that same version are used by all.